string wp
geometry_msgs/Pose pose
---
bool res
